app/frontend/src/features/communities/redirect.ts

```diff
@@ -1,8 +1,10 @@
 import { Page, PageType } from "pb/pages_pb";
-import { guideRoute, placeRoute } from "routes";
+import { routeToGuide, routeToPlace } from "routes";
 
 export const pageURL = (page: Page.AsObject) => {
```

Comment:
You can use shorter syntax with inline function:
export const pageURL = (page: Page.AsObject) => page.type === PageType.PAGE_TYPE_GUIDE
 ? routeToGuide(page.pageId, page.slug)
 :  routeToPlace(page.pageId, page.slug)